src/lib/browse/InterventionColorSelector.svelte

```diff
@@ -53,6 +31,72 @@
       color
     );
   }
+
+  function styleByFundingProgramme(): DataDrivenPropertyValueSpecification<string> {
+    let set: Set<string> = new Set();
+    for (let x of $schemes.values()) {
+      if (x.browse?.funding_programme) {
+        set.add(x.browse.funding_programme);
+      }
+    }
+    let programmes: string[] = Array.from(set);
+    programmes.sort();
+
+    let [colorMapping, returnedLegendRows] = getColorMappingAndLegend(
+      programmes,
+      colors.funding_programmes
+    );
+
+    let color = constructMatchExpression(
+      ["get", "funding_programme"],
+      colorMapping,
+      "grey"
+    );
+    legendRows = returnedLegendRows;
+    return color;
+  }
+
+  function styleByCurrentMilestone(): DataDrivenPropertyValueSpecification<string> {
+    let stageGates = [
+      "removed",
```

Comment:
I was just looking at the possible values here from the real output file, and they all match except for this -- I think it's called "superseded"